static/styles/zulip.scss

```diff
@@ -592,7 +592,7 @@ td.pointer {
     font-weight: 400;
     position: absolute;
     right: -105px;
-    line-height: 20px;
+    line-height: 18px;
```

Comment:
on adding background, message_time was not aligned with the action buttons' background

Reply:
Hmm, with 18px, the time text looks a little high to me; what do you think, @showell?

![image](https://user-images.githubusercontent.com/22248748/75191490-47d8ce80-5707-11ea-8ca6-0e65e04996d3.png)

vs. original, with 20px:

![image](https://user-images.githubusercontent.com/22248748/75191577-72c32280-5707-11ea-86fe-09e9ff24a623.png)

Is this the intended effect?